OpenRA.Mods.Common/Scripting/Properties/PlayerProperties.cs

```diff
@@ -30,9 +30,12 @@ public PlayerProperties(ScriptContext context, Player player)
 		[Desc("The player's color.")]
 		public HSLColor Color { get { return Player.Color; } }
 
-		[Desc("The player's race.")]
+		[Desc("The player's race. (DEPRECATED! Use the `Faction` property.)")]
 		public string Race { get { return Player.PlayerReference.Faction; } }
```

Comment:
Unfortunately, I doubt that mappers will read this notice.  Can you log a message to Game.Debug instead?

It might make sense for us to write a `[DeprecatedLuaAPI(message)]` attribute that automatically displays a message like "Warning: Map script is using a deprecated Lua API (PlayerProperties.Race).\n<message>".  That's a big scope creep for here though, so doing it manually is okay by me for now.